packages/shared/src/hooks/usePostById.ts

```diff
@@ -121,7 +121,7 @@ const usePostById = ({ id, options = {} }: UsePostByIdProps): UsePostById => {
   const {
     data: postById,
     isError,
-    isLoading,
+    isPending,
```

Comment:
I think only mutations got renamed no?
useQuery I thought was kept as `isLoading`?

If this is not the case we need to fix many more places.